test/uk/gov/hmrc/apiplatformdeskpro/controller/CreateTicketControllerSpec.scala

```diff
@@ -58,15 +58,17 @@ class CreateTicketControllerSpec extends AsyncHmrcSpec with StubControllerCompon
     val supportReason          = "supportReason"
     val teamMemberEmailAddress = "frank@example.com"
 
-    val createTicketRequest = CreateTicketRequest(
-      DeskproPerson(name, email),
-      subject,
-      message,
-      Some(apiName),
-      Some(applicationId),
-      Some(organisation),
-      Some(supportReason),
-      Some(teamMemberEmailAddress)
+    val createTicketRequestJson = Json.parse(
```

Comment:
Why are apiName, applicationId, organisation, supportReason and teamMemberEmailAddress no longer in the JSON?